### ISSUE TITLE
org-plus-contrib tried to get installed incorrectly from GNU Elpa instead of Org Elpa

### ISSUE THREAD

**Issue by kaushalmodi** [syl20bnr/spacemacs#9712]
#### Description :octocat:
Spacemacs attempts to install org-plus-contrib from incorrect Elpa, and then throws an error

#### Reproduction guide :beetle:
- Start Emacs
- Add `org` layer
- Save and restart

*Observed behaviour:* :eyes: :broken_heart:
```
--> installing package: evil-org@org... [1/12]
An error occurred while installing evil-org (error: (file-error https://elpa.gnu.org/packages/org-plus-contrib-20171009.tar Not found))
--> installing package: org-brain@org... [4/12]
An error occurred while installing org-brain (error: (file-error https://elpa.gnu.org/packages/org-plus-contrib-20171009.tar Not found))
```

*Expected behaviour:* :heart: :smile:
org-plus-contrib should be installed from Org Elpa, not GNU Elpa.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.0.60
- Spacemacs: 0.200.9
- Spacemacs branch: develop (rev. 07014de)
- Graphic display: t
- Distribution: spacemacs-base
- Editing style: emacs
- Completion: helm
- Layers:
```elisp
(helm emacs-lisp neotree
      org)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS NOTIFY ACL LIBSELINUX GNUTLS LIBXML2 FREETYPE LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK2 X11 MODULES

**Comment by bitmingw** [comment]
I got the similar issue that `org-plus-contrib` tar was not found from `https://elpa.gnu.org/packages`. That package should be downloaded from another source `https://orgmode.org/elpa`.

It turned out that for some reasons `https://orgmode.org/elpa` was not accessible from my network. After I set a proxy the problem was solved.

People who do have access to orgmode website may not be able to reproduce this issue.
